### ISSUE TITLE
Expiry field not getting Autofilled on iOS Chrome

### ISSUE THREAD

**Issue by jackocnr** [braintree/braintree-web#491]
### General information

* SDK version: 3.59.0
* Environment: Production
* Browser and OS: Chrome 80.0.3987.95 on iOS 13.4

### Issue description

If you tap the Card Number input and select your saved card, it successfully autofills the Card Number, but not the Expiry. This happens on our production website https://mubi.com/signup, and also on the Hosted Fields demo CodePen: https://codepen.io/braintree/full/zeamxM

Possibly related: https://github.com/braintree/braintree-web/issues/239

Do let me know if there's any more info I can provide. Thanks.

**Comment by crookedneighbor** [comment]
I've reproduced this. Thanks for the info. We'll have to determine if this is due to non-standard behavior in iOS Chrome specifically or Safari Webviews in general.

**Comment by andreymoskvenkov** [comment]
Does not work for us too. 
Any plans about fixing this issue? 
Visitors of our website are thinking, that this is our bug and sometimes are calling us with messages about that our website is buggy. Therefore, this is pretty important for us to get fixed.
https://www.scufgaming.com
Thanks!

**Comment by jackocnr** [comment]
Hi guys, it's been a few months - is there any update at all? Would be great to know if this is in the pipeline 🙏

**Comment by crookedneighbor** [comment]
This is in our backlog, but we have other issues we're working on that have priority right now. No ETA at this time.

**Comment by crookedneighbor** [comment]
A little update on this. I've figured out _why_ it isn't working.

Chrome for iOS does not use the underlying browser's built in autofill, it has its own custom implementation.

The way Hosted Fields is able to do autofill is that it has hidden inputs inside of the number frame that it sends to the corresponding visible fields in the other iframes. In order to make them not tabblable, we set those hidden inputs to have `tabindex=-1`, but chrome for iOS doesn't fill those inputs when they are not tabbable.

To get around this, we're detecting if the browser is chrome for ios and not setting the `tabindex` property. If the hidden inputs are tabbed to using the forward arrow on the software keyboard, it'll blur those inputs. To the user, it'll look like the keyboard was dismissed and they will be able to tap into the next field. (iOS based browsers don't allow programatic focussing, so currently, the customer would have to tap out of input or scroll to the next input anyway, so this should not cause much friction)

For users with autofill, the keyboard will be dismissed automatically once the autofilling is complete and they can scroll down to submit the form.

Will update here when a fix is released.
